suites/pre-release/dcf/4.preSignedURLTest.js

```diff
@@ -11,9 +11,23 @@ const {
   Gen3Response, requestUserInput,
 } = require('../../../utils/apiUtil');
 
+
 // Test elaborated for nci-crdc but it can be reused in other projects
 const TARGET_ENVIRONMENT = process.env.GEN3_COMMONS_HOSTNAME || 'nci-crdc-staging.datacommons.io';
 
+let filteredDIDs = {};
+
+const files = {
```

Comment:
you created this variable, but I don't see where you use it to create a new record

Reply:
This variable `files` is used in the Before block below in the script to create indexd record before every scenario and delete the record after each scenario.